app/models/queue_item.rb

```diff
@@ -0,0 +1,4 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user, foreign_key: :user_id
```

Comment:
Since you are following conventions here, you don't need to specify the `foreign_key` on either of these.